### PR TITLE
Issue 1 draft

### PR DESCRIPTION
**Pull request by dellsystem**
I wasn't sure how to incorporate some of the suggested links (some of which may be outdated), but feel free to add yours in directly to the document!

### CODE REVIEW

**Comment by jessesquires** [comment]
I can refine the spellchecker later so it's less noisy 😄

**Comment by jessesquires** [comment]
made a few commits to fix some issues 😄

**Comment by jessesquires** [comment]
built site locally. looks great to me!

File: _posts/2020-05-01-issue-1.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+<!--excerpt-->
+
+For many workers, the situation is intolerable, and they are taking a stand. Today, workers at Instacart, Shipt, Amazon warehouses, and your neighborhood Whole Foods are striking with a [list of demands](https://docs.google.com/document/d/1KD8uaVch14xBD1TTZ-TKbAub-IcqERS6O1OVeL2HbtY/edit). Christian Smalls, Amazon warehouse worker and organizer who was fired earlier this month, shared a [video montage](https://twitter.com/Shut_downAmazon/status/1255681771292721158) featuring dozens of essential workers who are walking out to protest for better working conditions.
+
+{% include post_image.html %}
+
+## THE WORKER’S PERSPECTIVE
```
Comment (jessesquires) [inline _posts/2020-05-01-issue-1.md:21]: I'm a fan of lowercase for all of these. Anyone else?

File: _posts/2020-05-01-issue-1.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+> Samy, an Iranian refugee who emigrated to California in the 2000s, described the terrible conditions that he and many of these “independent contractors” face. Unable to afford the Bay Area, he lives a hundred miles away from San Francisco, and rather than commute to the city daily, he sleeps in his car between shifts, sometimes not returning home for several days in a row. “We have no freedom,” he told me. “I sleep in my car. I eat in my car. I work in my car. That is not freedom. That is not flexibility.”
+
+## IN SONG
+
+We have not one but two (2!!) songs for you:
+
+### [Iceage - Lockdown Blues](https://iceage.bandcamp.com/track/lockdown-blues)
```
Comment (jessesquires) [inline _posts/2020-05-01-issue-1.md:43]: maybe h4 here? `####`

File: _posts/2020-05-01-issue-1.md (hunk 1 above)
Comment (dellsystem) [reply _posts/2020-05-01-issue-1.md:21]: yep make sense (i just copied from the doc), will do that now

**Comment by dellsystem** [comment]
i just made a bunch of changes responding to danny's comments in the doc (plus some other additions of my own). can someone take care of merging this? i have to be out of the house for the next couple hours

**Comment by anaulin** [comment]
Sure, @dellsystem I'll be around and online for the next little bit, if more help with this is needed.

**Comment by dellsystem** [comment]
also @stationaery I've made some edits to some of the copy you introduced - pls review!

**Comment by anaulin** [comment]
Also, it looks like maybe this branch needs rebasing, which I will do.

**Comment by anaulin** [comment]
I rebased this branch on `master` to get rid of some extraneous diffs for the header, and had to force-push-with-lease this branch. Anyone else working on this branch will need to re-pull before pushing, fyi @dellsystem @jessesquires .

**Comment by anaulin** [comment]
For reference, this is how this is looking like so far, once "compiled" into the site:
Screen Shot 2020-05-01 at 12 12 54 PM

**Comment by dellsystem** [comment]
oh damn i think that’s missing some of my local commits (incorporating
edits from the google doc) - not sure if those got lost or if i just didn’t
push then properly

**Comment by anaulin** [comment]
I was double-checking the links in the RSS feed, and it looks like markdown links with a relative URL (e.g. `[Restarting this newsletter](/about/)`) are showing up in the RSS feed also as relative, which I think might break once that gets sent out by MailChimp:
Screen Shot 2020-05-01 at 12 15 19 PM

@jessesquires do you know if there is a good way in Jekyll to tell it to prepend all relative links with `site.url`? (If not, for now I can just go and expand by hand all the links on this issue. But it would be nice to fix this throughout.)

**Comment by anaulin** [comment]
@dellsystem I can do a final check against the Google Doc before merging this in. Would that bring back those missing edits? (If not, how would you like to proceed?)

**Comment by dellsystem** [comment]
yep that should work, that would be great, thanks Ana! some of the links
may need to be converted to markdown if they’re not already

and good catch with the relative links, I believe jekyll has a tag for this
{% link %}

**Comment by stationaery** [comment]
I feel like the may day + fist image is less real/authentic than the tweet
screenshot- I added another option in the doc
<https://docs.google.com/document/d/12GU1GMzG-l4uSGDDI592tvZ7oaFn8XdWCzv6ljweorU/edit>.
Or at least, both?

**Comment by dellsystem** [comment]
the graphic is just a random one i whipped up, but i think it would be nice
to have the twc logo in there along with text explaining that the
newsletter has been rebooted (so people can understand what’s happening at
a glance)

maybe a screenshot of the tweet with the twc logo + newsletter reboot text
above or below it?

**Comment by stationaery** [comment]
I wouldn't want to put the TWC logo on Christian's tweet, especially since
it says MAY DAY PRESS RELEASE which links to a doc from his collective.

Also, the cover image is good and right but we're already swimming in red
and fists so let's nix the fist so the text stands out more and gets read.

And then let's use the tweet screenshot as-is.

Sound good?

**Comment by dellsystem** [comment]
i just checked and it seems like a lot of past issues use tweet screenshots
as images, so all good from me

**Comment by anaulin** [comment]
Looking into the behavior of the `link` Jekyll tag, it seems it is also creating relative links in the RSS feed (for reference: https://jekyllrb.com/docs/liquid/tags/#links). The only dirty-ish workaround I've found so far for this is doing something like:
```
[Restarting this newsletter]({{ site.url}}/about/)
```
I'll go ahead and edit the relative links to be in this ^ form, and if/when someone finds a better solution, we can tweak it further.

**Comment by anaulin** [comment]
I've updated this with the latest contents of the Google Doc, and the new header image. I now looks like:

Screen Shot 2020-05-01 at 1 34 13 PM

@stationaery can you scan through the content in this PR (handy direct link: https://github.com/techworkersco/techworkersco.github.io/pull/28/files?short_path=ff3e1ed#diff-ff3e1ed31dbb28123573aee776bcf3a5), and make sure that no edits are missing?

**Comment by jessesquires** [comment]
@dellsystem @anaulin 

yes, `[Restarting this newsletter]({{ site.url}}/about/)` is the best way to handle links. 

i'll add something to the contributing docs about this.

**Pull request merged**